requirements-dev.txt

```diff
@@ -1,12 +1,12 @@
-black[jupyter]>=23.3.0
+black[jupyter]==24.2.0
 konoha[janome]<6.0.0
 mypy>=1.2.0
 pytest>=7.3.1
-pytest-black-ng>=0.4.1
+pytest-black-ng==0.4.1
 pytest-github-actions-annotate-failures>=0.1.8
 pytest-mypy>=0.10.3
-pytest-ruff>=0.0.5
```

Comment:
Notice, that ruff itself is not pinned and pinning pytest-ruff is not enough for that.

So please add ruff as a version.
Also plead don't do hard pins but make softer ones:
`black[jupyter]==24.2.*` would allow us to accept bugfixes while requireing the dependency to behave the same.

Reply:
Thanks for the feedback! I've just updated the requirements file :)